src/briefcase/integrations/android_sdk.py

```diff
@@ -460,44 +472,32 @@ def select_target_device(self, device_or_avd):
         # Proces the user's choice
         if choice is None:
             # Create a new emulator. No device ID or AVD.
-            device = None
-            avd = None
-            name = None
+            device, avd, name = None, None, None
```

Comment:
Stylistically, I *really* dislike this pattern. One assigment, one line, unless you're actually unpacking a tuple.